flexget/plugins/internal/api_bluray.py

```diff
@@ -126,7 +126,11 @@ def __init__(self, title, year):
             # Used for parsing some more data, sadly with soup
             self.url = result['url']
 
-            movie_info_response = requests.get(self.url).content
+            try:
+                movie_info_response = requests.get(self.url).content
+            except (requests.RequestException, ConnectionError) as e:
+                raise LookupError('Couldn\'t connect to blu-ray.com. %s' % self.url)
```

Comment:
you should raise a `PluginError` exception here. Also, instead of escaping `'`, you can change the outer ones to `"`

Reply:
Done